homeassistant/components/homekit/type_sensors.py

```diff
@@ -53,23 +47,17 @@ def update_state(self, entity_id=None, old_state=None, new_state=None):
 class HumiditySensor(HomeAccessory):
     """Generate a HumiditySensor accessory as humidity sensor."""
 
-    def __init__(self, hass, entity_id, name, *args, **kwargs):
+    def __init__(self, entity_id, name, **kwargs):
```

Comment:
Also leave hass in here, and try to keep same signature as `__init__` in HomeAccessory (at the moment you swap name & entity_id).

Maybe keep name first, since `Accessory`'s signature is (name, **kwargs)

*same for other HomeAccesories